recipes/aghast/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "aghast" %}
+{% set version = "0.1.0rc4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/diana-hep/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: d458451970602df06722f388d6a2c4df2b83c149125fceea08bc3568e9ae3e1e
+  patches:
+    - 0001-Fix-packaging-for-conda.patch
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install ./python --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytest-runner
+  run:
```

Comment:
@jpivarski Am I missing any optional dependencies that you would like to have here?

When grepping I noticed `fnal_column_analysis_tools`, `pandas` and `root` are are imported.

Reply:
This is an open-ended set of things, so it wouldn't be a good precedent to start requiring them in the conda package. (This isn't like uproot, where there's a fixed set of things you might want, but I didn't want to include in the pip package.)

Also, `fnal_column_analysis_tools` isn't on PyPI, yet, I think.

Bottom line: I think it would be fine for the conda package to have exactly the same dependencies as the pip package. (Probably easier to maintain that way, too.)